core/docker/bin/run-trino

```diff
@@ -2,14 +2,9 @@
 
 set -xeuo pipefail
 
-set +e
-grep -s -q 'node.id' /etc/trino/node.properties
-NODE_ID_EXISTS=$?
-set -e
-
 NODE_ID=""
-if [[ ${NODE_ID_EXISTS} != 0 ]] ; then
+if ! grep -s -q 'node.id' /etc/trino/node.properties; then
     NODE_ID="-Dnode.id=${HOSTNAME}"
 fi
 
-exec /usr/lib/trino/bin/launcher run --etc-dir /etc/trino ${NODE_ID} "$@"
+exec /usr/lib/trino/bin/launcher run --etc-dir /etc/trino "${NODE_ID}" "$@"
```

Comment:
i think the quotes around ${NODE_ID} are actually incorrect

(to make it whitespace-awesome, you'd need to use arrays)

Reply:
It's a single switch.

Reply:
Huh you're right, it fails when `NODE_ID` is empty:
```
launcher: error: too many arguments
```
Can I add a comment about this?